networkx/algorithms/tests/test_dominance.py

```diff
@@ -261,3 +261,37 @@ def test_loops_larger(self):
                   'exit': set()}
         for n in df:
             assert_equal(set(df[n]),set(answer[n]))
+
+
+class TestDominatorTree(object):
+
+    def test_dominator_tree(self):
+        # from page 5 (labeled as 388 in bottom right)
+        # http://pages.cs.wisc.edu/~fischer/cs701.f08/lectures/Lecture19.pdf
+        edges = [(1, 2), (2, 3), (2, 4), (3, 5), (4, 5), (5, 6), (6, 7),
+                (7, 6), (7, 8)]
```

Comment:
The continuation line indentation does not match the line above it, one more space is needed (so the left parentheses line up). You may want to run [`flake8`][1] on your code to check for style issues like this.

[1]: http://flake8.pycqa.org/